core/common/src/main/java/alluxio/network/netty/NettyRPC.java

```diff
@@ -77,6 +73,19 @@ public void operationComplete(ChannelFuture future) throws Exception {
     return message;
   }
 
+  /**
+   * Sends a request and returns immediately without getting a response. Typically used for RPCs
+   * at best efforts (e.g., async cache).
+   *
+   * @param context the netty RPC context
+   * @param request the RPC request
+   */
+  public static void callWithoutBlocking(final NettyRPCContext context, ProtoMessage request)
+      throws IOException {
+    Channel channel = Preconditions.checkNotNull(context.getChannel());
+    channel.writeAndFlush(new RPCProtoMessage(request)).addListener(ChannelFutureListener.CLOSE);
```

Comment:
Is it good to close the channel, we will return it to the pool later right?

Reply:
fixed to `CLOSE_ON_FAILURE`, not `CLOSE`